financial-connections-example/gradle.properties

```diff
@@ -1 +1,2 @@
-STRIPE_ANDROID_NAMESPACE=com.stripe.android.financialconnections.example
\ No newline at end of file
+STRIPE_ANDROID_NAMESPACE=com.stripe.android.financialconnections.example
+android.enableR8.fullMode=true
```

Comment:
You shouldn't need this, it should already be handled by https://github.com/stripe/stripe-android/blob/000bc241b80912eac9ad61711ab237692b239774/gradle.properties#L16

Reply:
https://github.com/stripe/stripe-android/pull/6813/commits/a187372b29e3fe1a5585cc0a37c6050f0f3a9190